components/confirmation/Confirmation.js

```diff
@@ -46,14 +52,25 @@ export default class Confirmation extends Component {
     }
   }
 
+  keyExtractor = (item, index) => index.toString()
```

Comment:
Acho que colocar `_` no lugar de parametros não utilizados é melhor, mas talvez não fique tão claro caso futuramente queira usar essa função de outra forma...

```suggestion
  keyExtractor = (_, index) => index.toString()
```

Que tu acha?

Reply:
Nice! But we only do english here. So I couldn't understand a thing.

Just kidding. Spot on bro! 🌴 🏖